packages/astro/test/preview-routing.test.js

```diff
@@ -207,15 +205,15 @@ describe('Preview Routing', () => {
 				expect(response.status).to.equal(404);
 			});
 
-			it('404 when loading subpath root with trailing slash', async () => {
+			it('200 when loading subpath root with trailing slash', async () => {
```

Comment:
this is for `trailingSlash: never` right? I imagine that this should be a 404 and that the test below should be a 200.


I think the 200->404 change below is also incorrect

Reply:
This test has been updated.

Reply:
It still seems to suffer from the problem I pointed out above: If I set `trailingSlash: 'never',` I would expect `/blog/` to return a 404. Test seemed correct as written and incorrect with this change.

Reply:
ah got it, `/blog/` is the subpath root and not some page within the site. Missed that in the test description, LGTM!